regolith/helpers/l_abstracthelper.py

```diff
@@ -74,25 +82,37 @@ def sout(self):
         rc = self.rc
         presentations = self.gtx["presentations"]
 
+        if (not rc.author) and (not rc.year) and (not rc.loc_inst) and (not rc.title):
+            return
+
         for presentation in presentations:
             if rc.title is not None:
                 if rc.title.casefold() not in presentation.get('title').casefold():
                     continue
-            if rc.location is not None and 'location' in presentation:
-                if rc.location.casefold() not in presentation.get('location').casefold():
-                    continue
-            if rc.location is not None and 'location' not in presentation:
-                continue
+            if presentation['type'] == 'seminar' or presentation['type'] == 'colloquium':
```

Comment:
a better way to do this (less maintenance in general) is to make a global variable `SEMINAR_TYPES= ['seminar', colloquium]` at the top of the file and here you can type `if presentation.get('type') in SEMINAR_TYPES:` .  This way, if we decide later that "zoom-talk" is an allowed seminar type we just add it to that list and all the code just works.

Also, using `dict_object.get('item,on_fail)` method is a safer way to get a dict item.  If it can't find the dict item it will return whatever you put as the second argument, `on_fail` and the program won't crash.

Reply:
Thank you, I had no idea about dict_object.get('item,on_fail), this helped me cut down on some repetitive code. Sorry, java is what I know more formally so i'm slowly learning to be more formal programming in python as well since I am self-learning. ~work in progress